src/app/script/tabs_widget.h

```diff
@@ -0,0 +1,63 @@
+// Aseprite
+// Copyright (C) 2023  Igara Studio S.A.
+//
+// This program is distributed under the terms of
+// the End-User License Agreement for Aseprite.
+
+#ifndef APP_SCRIPT_TABS_H_INCLUDED
+#define APP_SCRIPT_TABS_H_INCLUDED
+#pragma once
+
+#include "app/ui/button_set.h"
+#include "ui/box.h"
+#include "ui/grid.h"
+#include "ui/separator.h"
+#include "ui/widget.h"
+
+namespace app {
+namespace script {
+
+class Tabs : public ui::VBox {
+
+public:
+  static ui::WidgetType Type();
+
+  Tabs(int selectorFlags);
+
+  app::ButtonSet::Item* addTab(ui::Grid* content);
+  void selectTab(int index);
+  void setSelectorFlags(int selectorFlags);
+
+  int tabIndexById(std::string text) const;
+  int tabIndexByText(std::string text) const;
+  std::string tabId(int index) const;
+  std::string tabText(int index) const;
+  int selectedTab() const;
+  int size() const { return m_pages ? m_pages->children().size() : 0; };
+
+  obs::signal<void(std::string deselTabId, std::string selTabId)> TabChanged;
```

Comment:
Probably I would just remove the signal parameters to simplify and access the tabs widget to get the selected tab:
```suggestion
  obs::signal<void()> TabChanged;
```